tools/update-undici.sh

```diff
@@ -26,6 +26,15 @@ rm -f deps/undici/undici.js
     "$NODE" "$NPM" install --global-style --no-bin-links --ignore-scripts undici
     cd node_modules/undici
     "$NODE" "$NPM" run build:node
+    # get the new version of undici
+    UNDICI_VERSION=$("$NODE" -p "require('./package.json').version")
+    # update this version information in src/node_metadata.cc
+    git show HEAD:src/node_metadata.cc | \
+      sed "s/UNDICI_VERSION \"[0-9.]*\"/UNDICI_VERSION \"$UNDICI_VERSION\"/" \
+      > "$ROOT/src/node_metadata.cc"
+    # commit these changes
+    git add "$ROOT/src/node_metadata.cc"
+    git commit -m "src: update undici version to $UNDICI_VERSION"
```

Comment:
```suggestion
```
This would actually get committed as a part of the update workflow in https://github.com/nodejs/node/blob/1a83ad6a693f851199608ae957ac5d4f76871485/.github/workflows/tools.yml#L135, so no need to do this here